### PR TITLE
Fix nits in DG

### PR DESCRIPTION
**Pull request by Khaleelur-Rahman**
Fix punctuation and line breaks in DG.

Update sequence diagrams.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -739,79 +767,79 @@ Return to [table of contents](#table-of-contents).
...
-* 1a. Any of the mandatory fields not specified
+* 1a. Any of the mandatory fields not specified.
 
-    * 1a1. JobFestGo informs user that mandatory fields not specified
+    * 1a1. JobFestGo informs user that mandatory fields not specified.
 
       Use case resumes at step 1.
 
-* 1b. Event name already exists.
+* 1b. Event name already exists
```
Comment (rionshocker) [inline docs/DeveloperGuide.md:786]: Missing a full-stop

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -739,79 +767,79 @@ Return to [table of contents](#table-of-contents).
...
 
     * 1b1. JobFestGo informs user that event name already exists.
 
       Use case resumes at step 1.
```
Comment (rionshocker) [inline docs/DeveloperGuide.md:791]: We are not including the case when event name is not alphanumeric?

**Review by rionshocker — CHANGES_REQUESTED** [review]
LGTM! Just some minor nits.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Khaleelur-Rahman) [reply docs/DeveloperGuide.md:786]: Good catch!

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (Khaleelur-Rahman) [reply docs/DeveloperGuide.md:791]: Yup, needs to be added. Added in the extension. Thanks for the reminder!

**Review by rionshocker — APPROVED** [review]
LGTM!

**Pull request merged**